internal/collections/map_test.go

```diff
@@ -108,21 +108,23 @@ func TestNewCaseSensitiveKeyMap(t *testing.T) {
 }
 
 func BenchmarkTxSetGet(b *testing.B) {
-	// Pre-generate keys in a map to avoid fmt.Sprintf overhead during benchmarking
 	keys := make(map[int]string, b.N)
 	for i := 0; i < b.N; i++ {
 		keys[i] = fmt.Sprintf("key%d", i)
 	}
 	c := NewCaseSensitiveKeyMap(variables.RequestHeaders)
+
+	for i := 0; i < b.N; i++ {
```

Comment:
Do we need this before the `Set` benchmark?

Reply:
Ah, you're right. was redundant, removed it.